administrator/components/com_admin/sql/updates/postgresql/5.1.1-2024-04-18.sql

```diff
@@ -0,0 +1,10 @@
+--
+-- Add a default value for the colorScheme in the Atum tenmplate on Joomla update
+-- only when a value is not already set.
+-- New installs will have the default value set in the installation sql.
+--
+
+UPDATE "#__template_styles"
+SET "params" = jsonb_set("params", '{colorScheme}', '"os"', true)
+WHERE "template" = 'atum'
+AND NOT ("params" -> 'colorScheme' ? 'os');
```

Comment:
```suggestion
AND "params"::jsonb->>'colorScheme' IS NULL;
```
We should explicitly cast the "params" column to type "jsonb", use the "->>" operator to get a result of type "text" and not "->" which would return a json type, and there must be no space between the database column, the operator and the json field.